<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-03-28T14:16:34Z" name="Suite" finished-at="2021-03-28T14:17:06Z" duration-ms="32551">
    <groups>
    </groups>
    <test started-at="2021-03-28T14:16:34Z" name="Test" finished-at="2021-03-28T14:17:06Z" duration-ms="32551">
      <class name="com.Test.GW_BillingCenter_Test1">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.Test.GW_BillingCenter_Test1@47c81abf]" started-at="2021-03-28T19:46:34Z" name="setUp" finished-at="2021-03-28T19:46:39Z" duration-ms="5443" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="BillingTest()[pri:0, instance:com.Test.GW_BillingCenter_Test1@47c81abf]" started-at="2021-03-28T19:46:39Z" name="BillingTest" finished-at="2021-03-28T19:47:05Z" duration-ms="26433" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Automation9] but found [Automation10]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Automation9] but found [Automation10]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at com.pages.Sandbox.BillingCenter_Resuables.bc_AccountSummary(BillingCenter_Resuables.java:195)
	at com.Test.GW_BillingCenter_Test1.BillingTest(GW_BillingCenter_Test1.java:244)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BillingTest -->
        <test-method is-config="true" signature="Closure()[pri:0, instance:com.Test.GW_BillingCenter_Test1@47c81abf]" started-at="2021-03-28T19:47:05Z" name="Closure" finished-at="2021-03-28T19:47:06Z" duration-ms="658" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Closure -->
      </class> <!-- com.Test.GW_BillingCenter_Test1 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
